### DISCUSSION TITLE
How to Install on Garuda Linux

### DISCUSSION THREAD

**Discussion by boundingbox** [OpenTabletDriver/OpenTabletDriver#2108] [General]
This is for those who are less familiar with Arch's nuts and bolts, it's just more granular for those who might need more clear directions like myself. I used Trizen to install. Basically, it's an anti-frustration guide. The final commands are taken from the vanilla Arch directions.

Open konsole/Fish, NOT Octopi.
Install Trizen or Pacman, under sudo: 
```sudo pacman -s Trizen```
Run Trizen/Pacman WITHOUT sudo perms. (It'll prompt you for password later. It doesn't let you do this for a reason!)
```trizen -s OpenTabletDriver```
Run through the prompts, answer N to review/edit, Y to install it and dependencies.
Run these commands:
```systemctl --user daemon-reload```
```systemctl --user enable opentabletdriver --now```

**Comment by ghost** [comment]
```
sudo pacman -S base-devel git
git clone https://aur.archlinux.org/opentabletdriver.git
cd opentabletdriver
makepkg -si
```
for installing the package this works

**Comment by jamesbt365** [comment]
Garuda is arch based, therefore uses the same exact instructions as arch. What this "guide" says is how to install an aur helper, then install otd through it.
